0_config_rmk.yml

```diff
@@ -0,0 +1,31 @@
+target_default: 0_config
+
+packages:
+  - sp
+  - scipiper
+  - dplyr
+
+sources:
+  - 0_config/code/nldas_cube_utils.R
+targets:
+  0_config: 
+    depends: 
+      - 1_nldas_cubes_rmk.yml
+  
+  nldas_time_range:
+    command: list(t0=I(0), t1 = I(12))
+    
+  nldas_time_stride:
+    command: c(I(10))
+    
+  nldas_cells:
+    command: data.frame(x=I(c(231,239,223)),y=I(c(135,135,141))) # will come from code or spatial data...
+    
+  nldas_cube:
+    command: nldas_cube_diff(nldas_cells, '0_config/in/nldas_cells.rds')
```

Comment:
and would use `I()` around the 2nd arg, right? so that merely generating some feather files wouldn't force this command to come up with a new set of cubes?